compact/far_backward/vacuum.xml

```diff
@@ -22,7 +22,7 @@
       <dimensions xL="Beam_WidthL" xR="Beam_WidthR" y="Beam_Height" z="Beam_Length"/>
 
       <comment> Vacuum volume joining with lumi </comment>
-      <exitdim    x="Exit_Width"   y="Exit_Height"  lumiZ="LumiWin_Z" lumiR="LumiWin_R" maxTheta="Exit_Theta"/>
+      <exitdim    x="Exit_Width"   y="Exit_Height"  lumiZ="LumiWin_Z" lumiR="LumiWin_thickness" maxTheta="Exit_Theta"/>
```

Comment:
This currently defines the radius of a circular hole in the steel beampipe as a joining position to the lumi system so LumiWin_thickness is the wrong variable here. I anticipated the lumi-description having an independent extension of the vacuum pipes from this point.

Reply:
With the latest ideas from Kryzsztof, we won't have an extra vacuum system for the lumi system.  But ok, I can put LumiWin_R back in.

Reply:
Will there not at least need to be a separate vacuum spanning between the convertor and tracking detectors?

Reply:
I doubt it.  That will be expensive and technically challenging compared to the "sweeper magnet" idea.